docs/oauth.md

```diff
@@ -202,6 +202,62 @@ To log out of id.webmaker.org, use the /logout route
 After a successfully clearing the session, you'll be redirected to the redirect_uri of the supplied client_id, with a `logout=true` in the search portion of the redirect URL
 
 e.g `https://example.org/oauth_redirect?logout=true`
+
+## Native Application Flow
+
+Native applications can use the password grant type to authenticate Webmaker users, without compromising their client_secret.
```

Comment:
Should we do something like remove CORS headers on the `/login/oauth/access_token` route to ensure this?